### PR TITLE
[TIMOB-20037] Android: Wrong Ti.Android.currentActivity in `open` handler

### PR DESCRIPTION
**Pull request by ypbnv**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-20037

**Test case:**
```
var window = Ti.UI.createWindow({backgroundColor:'#F00'});

window.addEventListener('open', function (e) {
    Ti.API.debug('Main window opened', e.source.getActivity() === Ti.Android.currentActivity);
});

window.open();
```

**Optional Description:**
Added Kroll property getter in the module.

@jquick-axway I have missed or messed up something here. Building the SDK with the changes failed after a second attempt. I will update you once I find out what exactly happened.

Update: It was a problem with my setup. The code should work fine.

### CODE REVIEW

File: android/titanium/src/java/org/appcelerator/titanium/TiApplication.java [hunk 1]
```diff
@@ -57,9 +18,32 @@
 import android.support.multidex.MultiDex;
 import android.util.DisplayMetrics;
 import android.view.accessibility.AccessibilityManager;
-
 import com.appcelerator.aps.APSAnalytics;
 import com.appcelerator.aps.APSAnalytics.DeployType;
+import org.appcelerator.kroll.*;
```
Comment (garymathews) [inline android/titanium/src/java/org/appcelerator/titanium/TiApplication.java]: Should revert these imports back to not use wildcards

**Review by garymathews — CHANGES_REQUESTED** [review]
CR: Fix imports

File: android/modules/android/src/java/ti/modules/titanium/android/AndroidModule.java [hunk 2]
```diff
@@ -325,6 +326,11 @@ public RProxy getR() {
 		return r;
 	}
 
+	@Kroll.method @Kroll.getProperty
+	public ActivityProxy getCurrentActivity() {
+		return TiApplication.getAppCurrentActivity().getActivityProxy();
```
Comment (garymathews) [inline android/modules/android/src/java/ti/modules/titanium/android/AndroidModule.java]: It is possible for `getAppCurrentActivity()` to return `null`, best add a guard.

**Comment by ypbnv** [comment]
@garymathews Reverted the imports and added a guard and a warning.

**Review by garymathews — APPROVED** [review]
CR: **PASS**

**Review by ssjsamir — APPROVED** [review]
FR passed: currentActivity is now the same as the current window activity. This can be seen by the following output in the console: 
```
[DEBUG] :  Window: Checkpoint: postWindowCreated()
[DEBUG] :  Main window opened true
```
**Test Steps**
+ Downloaded the SDK Build form this PR
+ Created a new Titanium project
+ Copied the the test case from description
+ Ran the application 
+ Saw the above output in the console

**Test Environment**
Appcelerator Command-Line Interface, version 6.2.2
Google Nexus 6P (7.1.1)
Operating System Name: Mac OS X El Capitan
Operating System Version: 10.11.6
Node.js Version: 6.10.1
Xcode: 8.2
Appcelerator Studio: 4.9.0.201705251638

**Pull request merged**